CryptoPkg/Library/BaseCryptLibOnOneCrypto/DxeCryptLib.c

```diff
@@ -0,0 +1,49 @@
+/** @file
+  Locates and sets the global gCryptoProtocol pointer to the OneCrypto DXE Protocol.
+
+  Copyright (c) Microsoft Corporation.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+#include <PiDxe.h>
+#include <Library/BaseLib.h>
+#include <Library/DebugLib.h>
+#include <Library/UefiBootServicesTableLib.h>
+#include <Protocol/OneCrypto.h>
+
+extern ONE_CRYPTO_PROTOCOL  *gCryptoProtocol;
+
+/**
+  Locate the valid Crypto Protocol.
+
+  @param  ImageHandle   The firmware allocated handle for the EFI image.

```

Comment:
nit: inconsistency in documentation. All of the one crypt header functions have `@param[out]` or `@param[in]` but this does not. I cannot remember which one is the new way and which one is the old way. But should definitely use the new way.